data/json/furniture.json

```diff
@@ -3251,6 +3262,7 @@
     "type": "furniture",
     "id": "f_ergometer",
     "name": "ergometer",
+    "description": "An exercise rowing machine.  Without power, it can no longer help you regulate your workout.  Might have useful electronic parts in it.",
```

Comment:
```suggestion
    "description": "A rowing machine for exercising.  Without power, it can no longer help you with your workout.  Might have useful electronic parts in it.",
```